hw/ip/hmac/data/hmac.hjson

```diff
@@ -316,6 +316,19 @@
           tags: [// Exclude fields with non-zero reset value from write-read checks.
                  "excl:CsrNonInitTests:CsrExclWriteCheck"]
         }
+        { bits: "14",
```

Comment:
I had also just moved this now (in my next push) after `digest_swap` such that they are `endian_swap`, `digest_swap` and `key_swap` in sequence. @andreaskurth

Reply:
That makes sense, thx!